integration-cli/docker_cli_rmi_test.go

```diff
@@ -77,6 +77,37 @@ func TestRmiTag(t *testing.T) {
 	logDone("rmi - tag,rmi- tagging the same images multiple times then removing tags")
 }
 
+func TestRmiImgIDForce(t *testing.T) {
+	d := NewDaemon(t)
+
+	if err := d.StartWithBusybox(); err != nil {
+		t.Fatal(err)
+	}
+	imagesBefore, _ := d.Cmd("images", "-a")
+	d.Cmd("tag", "busybox", "utest:tag1")
```

Comment:
It might be best to check the errors of these commands. Not sure if the results below are viable if these all result in an unchecked error.